### PR TITLE
Added int to days TimeInterval helper.

### PR DESCRIPTION
**Pull request by elland**
Should simplify dealing with time boundary checks such as "is this older than X days".

```swift
if data.creationDate.timeIntervalSinceNow >= 8.days { return "Is more than a week old" }
```

### CODE REVIEW

File: Sources/Int+Date.swift [hunk 1]
```diff
@@ -19,6 +19,6 @@ extension Int {
     /// Returns a TimeInterval for an approximate number of days.
     /// Ex: 3.days == 259200 seconds. Of courrse is not Date/Time aware. Use it with care.
```
Comment (designatednerd) [inline Sources/Int+Date.swift]: Nit: `courrse` -> `course`

**Review by designatednerd — CHANGES_REQUESTED** [review]
So my objection to this is more from a standpoint of "I know we say this is only for approximate math but we should really be encouraging consumers of this library to use exact date math, not time intervals."

What about an extension on NSDate that allows you to get the NSDateComponents between two dates? I'm  basically thinking something like `dateComponentsSince:` and `dateComponentsSinceNow`, which returns the `NSDateComponents` between two dates. 

You might have to pass in a default calendar in the function so people can do math on whatever calendar they want, but for the most part that should be `current` anyway. 

The other advantage of this is that then you can check any unit of the components (`day`, `month`, `year`) without having to write additional extension methods for them.

**Comment by 3lvis** [comment]
Isn't doing something like this better?

screen shot 2018-04-10 at 14 52 04

**Comment by designatednerd** [comment]
@3lvis The third bullet point on that slide indicates it's not for use for comparing dates

**Comment by designatednerd** [comment]
(meaning, you can say "is date x within the unit granularity of seconds / minutes / hours of date y", but it doesn't tell you how far date x is from date y, which i think is what @Elland is trying to calculate here)

**Comment by 3lvis** [comment]
I see, thanks for clarifying :)

**Comment by elland** [comment]
Gonna use `Calendar` and `DateInterval` instead.

**Pull request closed without merging**